pipelines/build/common/config_regeneration.groovy

```diff
@@ -609,11 +607,11 @@ class Regeneration implements Serializable {
 
                     // Parse api response to only extract the relevant pipeline
                     getPipelines.jobs.name.each { pipeline ->
-                        if (pipeline == "openjdk${versionNumbers[0]}-pipeline") {
+                        if (pipeline == (jobType != "prototype" ? "openjdk${versionNumbers[0]}-pipeline" : "prototype-openjdk${versionNumbers[0]}-pipeline")) {
```

Comment:
I'd extract the (jobType != "prototype" ? "openjdk${versionNumbers[0]}-pipeline" : "prototype-openjdk${versionNumbers[0]}-pipeline") to a local variable and check against that.